source/fundamentals/crud/read-write-pref.txt

```diff
@@ -0,0 +1,252 @@
+
+For more information, see :manual:`Read Concern </reference/read-concern/>`. For more detailed API descriptions, see `API ReadConcern <{+api+}/classes/ReadConcern.html>`__
+
+Constructor
+~~~~~~~~~~~
+
+A new ``ReadConcern`` takes a ``ReadConcernLevel`` string as a parameter. 
+
+.. code-block:: js
+
+   new ReadConcern(readConcernLevel): ReadConcern
+   
+The levels are defined by an enumerable set of the following options:
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - ReadConcernLevel
+     - Description
+
+   * - ``'local'``
+     - | The query returns the instance’s most recent data.
+       | For more information, see the :rapid:`Read Concern specification </reference/read-concern-local/#mongodb-readconcern-readconcern.-local->`.
+
+   * - ``'available'``
+     - The query returns data from the instance with no guarantee that the data has been written to a majority of the replica set members. 
+       | For more information, see the :rapid:`Read Concern specification </reference/read-concern-available/#mongodb-readconcern-readconcern.-available->`.
+
+   * - ``'majority'``
+     - | The query returns the instance’s most recent data acknowledged as having been written to a majority of members in the replica set.  
+       | For more information, see the :rapid:`Read Concern specification </reference/read-concern-majority/#mongodb-readconcern-readconcern.-majority->`.
+
+   * - ``'linearizable'``
+     - | The query returns data that reflects all successful writes issued that had a write concern of ``majority`` and were acknowledged before the start of the read operation. 
+       | For more information, see the :rapid:`Read Concern specification </reference/read-concern-linearizable/#mongodb-readconcern-readconcern.-linearizable->`.
+
+   * - ``'snapshot'``
+     - | The query returns a complete copy of the data in a ``mongod`` instance at a specific point in time. Only available for operations within multi-document transactions.  
+       | For more information, see the :rapid:`Read Concern specification </reference/read-concern-snapshot/#mongodb-readconcern-readconcern.-snapshot->`.
+
+Specify a ``ReadConcern`` for a collection
+``````````````````````````````````````````
+
+This code specifies the read concern level of an existing collection to ``'majority'``:
+
+.. code-block:: js
+
+   rc = new ReadConcern('majority')
+   db.collection.find({}).options{ readConcern: rc }
+
+Read Preference
+---------------
+
+The read concern determines how queries are routed, that is which member of a replica set is read. The default mode is to read from the primary member of a replica set.
```

Comment:
Suggestion: 
I think this is correctly identifying a [style guide rule](https://www.mongodb.com/docs/meta/style-guide/writing/use-that-which-correctly/). Normally, you'd use "which" to specify further detail and use "that" to specify a consequence (restrictive).

I think it could be made more concise and avoid the restrictive clause, e.g.:
"The read concern determines which members of a replica set are read when performing queries."

Reply:
Originally I was using "i.e." to further describe the routing, but Snooty told me to change it to "that is", and then gave me this error. I like your suggestion tho, so I'll make that change. 

But in the case where I wanted to keep both phrasings, I think the comma is necessary.